CONTRIBUTORS.txt

```diff
@@ -34,9 +38,14 @@ needs to be listed here.
 - Lev Walkin <vlm@lionet.info>
 - Ludovic Dewailly <me@ldewailly.com>
 - Matt Eaton <agnosticdev@gmail.com>
+- Max Desiatov <max.desiatov@gmail.com>
```

Comment:
can you please change this to max@desiatov.com? Thanks!

Reply:
Done.